CHANGES.md

```diff
@@ -24,6 +24,8 @@
 - Only detect internal OCaml tools with `.opt` extensions. Previously, this
   detection applied to other binaries as well (@kit-ty-kate, @rgrinberg, #3051).
 
+- Never filter-out vendored packages (#3705, @diml)
```

Comment:
Something like this is more clear: Correctly build vendored packages in `-p` mode. These packages were incorrectly filtered before.

Reply:
Indeed, I updated the message